aws-api/src/androidTest/assets/create-event.graphql

```diff
@@ -1,5 +1,5 @@
 mutation CreateEvent($name: String!, $when: String!, $where: String!, $description: String!) {
-  createEvent(name: $name, when: $when, where: $where, description: $description) {
+  createEvent(input: {name: $name, when: $when, where: $where, description: $description}) {
```

Comment:
I think this is usually supposed to be a `CreateEventInput!` object, but if it's working as is, maybe I'm wrong?

```suggestion
  createEvent(input: CreateEventInput!) {
```

Also, if you make this change, don't you also need to update the code in `GraphQLInstrumentationTest::createEvent` to add an "input" variable, instead of the other 4 variables (name, when, where, description)?

Reply:
I was confused by this as well. I think the API that used to be hit by this test was created via the AppSync console using one of the sample apps they provide, so it differs from the schema that gets generated by the Amplify CLI. Since we're now generating via the CLI, some of the calls had to be tweaked a bit.

At first, I tried adding "input" to the call from `GraphQLInstrumentationTest::createEvent`, but turns out that those parameter names have to match what's specified in the mutation declaration `mutation CreateEvent($name: String!, $when: String!, $where: String!, $description: String!)`

Reply:
Also, `CreateEventInput` doesn't exist in the context of this query. I think it would have to be declared as part of the `create-event.graphql` file.

Reply:
Got it, thanks for the explanation!